SnakesAndLadder/src/services/GameService.java

```diff
@@ -0,0 +1,45 @@
+package services;
+
+import models.*;
+import strategies.IWinningStrategy;
+
+import java.util.List;
+import java.util.Random;
+
+public class GameService {
+    IWinningStrategy winningStrategy = null;
```

Comment:
Shouldn't this winning strategy be kept at the game level(inside game model) so that different games can have different winning strategies